PWGCF/Flow/Tasks/flowGfwTask.cxx

```diff
@@ -52,6 +57,20 @@ using namespace o2::aod::evsel;
 
 #define O2_DEFINE_CONFIGURABLE(NAME, TYPE, DEFAULT, HELP) Configurable<TYPE> NAME{#NAME, DEFAULT, HELP};
 
+namespace o2::aod
+{
+using SimCollisions = soa::Join<aod::Collisions, aod::EvSels, aod::Mults,
+                                aod::McCollisionLabels, o2::aod::CentFT0Cs,
+                                aod::CentFT0As, aod::CentFT0Ms, aod::CentFV0As, aod::CentFT0CVariant1s>;
+
+using SimTracks = soa::Join<aod::Tracks, aod::TrackSelection, aod::TracksExtra,
+                            aod::TracksDCA, aod::McTrackLabels>;
+} // namespace o2::aod
+
+using Colls = soa::Filtered<soa::Join<aod::Collisions, aod::EvSels, aod::Mults, aod::CentFT0Cs, aod::CentFT0As, aod::CentFT0Ms, aod::CentFV0As, aod::CentFT0CVariant1s>>;
+using AodTracks = soa::Filtered<soa::Join<aod::Tracks, aod::TrackSelection, aod::TracksDCA, aod::TracksExtra>>;
+using BCsRun3 = soa::Join<aod::BCs, aod::Timestamps, aod::BcSels, aod::Run3MatchedToBCSparse>;
```

Comment:
Why incorporating `using` clauses to the standard `o2::aod` `namespace`?
`namespace` are for avoiding name clashes so, if needed, you should use your own `namespace`, for instance, `flowgfw`
Please, have it in mind for the next iteration